src/guides/awaitedCommands.md

```diff
@@ -2,67 +2,148 @@
 
 Awaited commands are a special type of command where the bot waits for the user's response.
 
-![end result](https://i.imgur.com/rRRcIXA.jpg) 
-
-> In the above screenshot, the bot asks the user to provide a number and then checks, if provided value is a number.
-
-
-## Getting Started
-To create an awaited command, following function and callbacks are used -
-
-### $awaitFunc[]
-`$awaitFunc` is a function. It's used to initiate an awaited command.
-
-#### Usage
+## Content
+[**Used**](#used) > [**Supported Filters**](#supported-filters) > [**$awaitFunc[]**](#awaitfunc) > [**$awaitedCommand[]**](#awaitedcommand) > [**$awaitedCommandError[]**](#awaitedcommanderror) > [**Create Await command**](#create-await)
+
+## Used
+- [`$awaitFunc[]`](../bdscript/awaitFunc.md)
+- [`$awaitedCommand[]`](../callbacks/awaitedCommand.md)
+- [`$awaitedCommandError[]`](../callbacks/awaitedCommandError.md)
+## Supported Filters
+- `<numeric>`: Accepts only number input.
+- `<word1/word2/...>`: Accepts only specified words provided inside `<>`. Use `/` as a separator for multiple words.
+
+# $awaitFunc
+Used to initiate an awaited command.
+## Syntax
+```
+$awaitFunc[name;(user ID;channel ID)]
 ```
-$awaitFunc[command name;(user ID;channel ID)]
+### Parameters
+- `command name` `(Type: String || Flag: Required)`: The name used inside [`$awaitedCommand[]`](#awaitedcommand) and [`$awaitedCommandError[]`](#awaitedcommanderror) callbacks.
+- `user ID` `(Type: Snowflake || Flag: Vacantable)`: The user the awaited command will trigger for. Uses command author, if `user ID` is not given.
+- `channel ID` `(Type: Snowflake || Flag: Optional)`: The channel where the command will be awaited. Uses current channel, if `channel ID` is not given.
+## Example
 ```
-- `command name` - The name which is used inside [`$awaitedCommand`](#awaitedcommand) and [`$awaitedCommandError`](#awaitedcommanderror) callback.
-- `user ID` - The user the awaited command will trigger for. Uses command author, if `user ID` is not given.
-- `channel ID` - The channel id where the command should be awaited. Uses current channel, if `channel ID` is not given.
+$nomention
+What do you want me to say?
+$awaitFunc[say]
+```
+![example](https://user-images.githubusercontent.com/113303649/212294420-acf01905-c9f5-4673-99f0-375f9d786f25.png)
 
-### $awaitedCommand[]
-`$awaitedCommand` is a callback. It gets triggered when an awaited command gets initiated.
+# $awaitedCommand
+_Triggered when an awaited command gets initiated._
```

Comment:
This callback only runs after `$awaitFunc[]` was run and the given user sent a message in the given channel aka responded. So calling this initiation is not correct.
```suggestion

# $awaitedCommand
_Triggered when an awaited command gets responded to._
```